src/4.3_to_5.0/migrateKpi.ts

```diff
@@ -206,6 +210,25 @@ export function migrateKpi(
     }),
   };
 
+  try {
+    if (legacyMdx) {
+      // Migrate manually entered KPI titles.
+      const migratedTitles = getMigratedKpiTitles(legacyKpiState, {
+        cube,
+        mapping,
+        legacyMdx,
+      });
+      migratedWidgetState.titles = migratedTitles;
+    }
+  } catch (error) {
+    // Migrating the KPI titles is a best effort.
+    // The migration script should not fail if this part errors.
+    throw new PartialMigrationError(
+      `Could not migrate the titles of the featured values widget named "${legacyKpiState.name}"`,
+      serializeWidgetState(migratedWidgetState),
+    );
```

Comment:
I think it'd be helpful to preserve the error message when wrapping it in `PartialMigrationError`, to help consumers of the CLI fix that error.